workspaces/feedback/plugins/feedback-backend/dist-dynamic/package.json

```diff
@@ -60,10 +60,11 @@
     "plugin"
   ],
   "homepage": "https://red.ht/rhdh",
-  "bugs": "https://github.com/janus-idp/backstage-plugins/issues",
+  "bugs": "https://github.com/backstage/community-plugins/issues",
   "maintainers": [
-    "@janus-idp/maintainers-plugins",
-    "@janus-idp/devex-uxe"
+    "riginoommen",
+    "deshmukhmayur",
```

Comment:
are these people also in the https://github.com/backstage/community-plugins/blob/main/.github/CODEOWNERS file? if not please add that to this PR

Reply:
Those people have been added to the codeowners file on the other pr https://github.com/backstage/community-plugins/pull/1101